modules/ui.styl

```diff
@@ -75,6 +75,7 @@
     height: 1cap
     display: inline-block
     margin-right: .25rem
+    margin-bottom: 2px
```

Comment:
It seems likely that this value needs to be a relative not absolute measure. The height of the dot is `1cap` which off the top of my head is a measure of the cap height of the current font at whatever size that font is. Probably this `2px` needs to be converted to a decimal `cap` value. You could work that out via inspector from whichever location on the site you've measured the `2px` difference